lib/easypost.rb

```diff
@@ -52,10 +52,17 @@ class << self
 
   self.api_base = DEFAULT_API_BASE
 
+  def self.user_agent
+    @user_agent ||=
```

Comment:
Question: Why are we assigning this to a variable? In Ruby, the last line of a function is what gets returned. We can inherently have the string be returned (this should really be a constant) since the function isn't actually doing anything helpful here.